nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/inotify/EventAttributes.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.processors.hadoop.inotify;
+
+final class EventAttributes {
+    static final String EVENT_PATH = "hdfs.inotify.event.path";
+    static final String EVENT_TYPE = "hdfs.inotify.event.type";
+    static final String MIME_TYPE = "mime.type";
```

Comment:
The reason I did this is so that I wouldn't have to hardcode a string in the WriteAttribute annotation and then use the CoreAttributes and EventAttributes when writing said attributes. I thought it would be simpler to consolidated into one class. If it is preferred to use CoreAttributes I can switch mime.type to do so.